rmc/utils/mpc.py

```diff
@@ -432,20 +433,57 @@ def _run_glm(
         min_single_aic_var,
     )
 
+    # List possible variable combinations for Joint Regression
+    comb_var = []
```

Comment:
Suggestion for more intuitive var name
```suggestion
    var_combs = []
```